cluster/kube-util.sh

```diff
@@ -14,118 +14,46 @@
-
-# Must ensure that the following ENV vars are set
-function detect-master {
-	echo "KUBE_MASTER_IP: $KUBE_MASTER_IP" 1>&2
-	echo "KUBE_MASTER: $KUBE_MASTER" 1>&2
-}
-
-# Get node names if they are not static.
-function detect-node-names {
-	echo "NODE_NAMES: [${NODE_NAMES[*]}]" 1>&2
-}
-
-# Get node IP addresses and store in KUBE_NODE_IP_ADDRESSES[]
-function detect-nodes {
-	echo "KUBE_NODE_IP_ADDRESSES: [${KUBE_NODE_IP_ADDRESSES[*]}]" 1>&2
-}
-
-# Verify prereqs on host machine
-function verify-prereqs {
-	echo "TODO: verify-prereqs" 1>&2
-}
-
-# Validate a kubernetes cluster
-function validate-cluster {
-	# by default call the generic validate-cluster.sh script, customizable by
-	# any cluster provider if this does not fit.
-	"${KUBE_ROOT}/cluster/validate-cluster.sh"
-}
-
-# Instantiate a kubernetes cluster
-function kube-up {
-	echo "TODO: kube-up" 1>&2
-}
-
-# Delete a kubernetes cluster
-function kube-down {
-	echo "TODO: kube-down" 1>&2
-}
-
-# Update a kubernetes cluster
-function kube-push {
-	echo "TODO: kube-push" 1>&2
-}
-
-# Prepare update a kubernetes component
-function prepare-push {
-	echo "TODO: prepare-push" 1>&2
-}
-
-# Update a kubernetes master
-function push-master {
-	echo "TODO: push-master" 1>&2
-}
-
-# Update a kubernetes node
-function push-node {
-	echo "TODO: push-node" 1>&2
-}
-
-# Execute prior to running tests to build a release if required for env
-function test-build-release {
-	echo "TODO: test-build-release" 1>&2
-}
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
-# Execute prior to running tests to initialize required structure
-function test-setup {
-	echo "TODO: test-setup" 1>&2
-}
+source "${KUBE_ROOT}/cluster/skeleton/util.sh"
 
-# Execute after running tests to perform any required clean-up
-function test-teardown {
-	echo "TODO: test-teardown" 1>&2
-}
-
-KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBERNETES_PROVIDER="${KUBERNETES_PROVIDER:-gce}"
 PROVIDER_UTILS="${KUBE_ROOT}/cluster/${KUBERNETES_PROVIDER}/util.sh"
 if [ -f ${PROVIDER_UTILS} ]; then
-    source "${PROVIDER_UTILS}"
+	source "${PROVIDER_UTILS}"
 fi
 
 # Federation utils
 
 # Should NOT be called within the global scope, unless setting the desired global zone vars
 # This function is currently NOT USED in the global scope
 function set-federated-zone-vars {
-    zone="$1"
-    export OVERRIDE_CONTEXT="federation-e2e-${KUBERNETES_PROVIDER}-$zone"
-    echo "Setting zone vars to: $OVERRIDE_CONTEXT"
-    if [[ "$KUBERNETES_PROVIDER" == "gce"  ]];then
+	zone="$1"
+	export OVERRIDE_CONTEXT="federation-e2e-${KUBERNETES_PROVIDER}-$zone"
+	echo "Setting zone vars to: $OVERRIDE_CONTEXT"
+	if [[ "$KUBERNETES_PROVIDER" == "gce"  ]];then
 
-	export KUBE_GCE_ZONE="$zone"
-	# gcloud has a 61 character limit, and for firewall rules this
-	# prefix gets appended to itslef, with some extra information
-	# need tot keep it short
-	export KUBE_GCE_INSTANCE_PREFIX="${USER}-${zone}"
+		export KUBE_GCE_ZONE="$zone"
```

Comment:
These are just converting from a mix of space/tab to tab only. So:

```
function set-federated-zone-vars {
\tif [[ "$KUBERNETES_PROVIDER" == "gce"  ]];then
\t\texport KUBE_GCE_ZONE="$zone"
```

Did you want something else? Also, I'm fine with dropping the whitespace changes from this PR (unrelated). I was just already in the file and the space/tab mixed block rendered weird for me.